fixture/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[workspace]
+members = [
```

Comment:
One integrity zome and two coordinators so that we can set up tests which have the same DNA hash but are actually different apps